pkg/sensors/accelerator/device/sources/nvml.go

```diff
@@ -28,23 +28,22 @@ import (
 
 	"github.com/sustainable-computing-io/kepler/pkg/config"
 	_dev "github.com/sustainable-computing-io/kepler/pkg/sensors/accelerator/device"
```

Comment:
There's no conflict here so no need to rename the import

Reply:
this was mainly to avoid the clash in the 1st param of:

```
func (n *GPUNvml) ProcessResourceUtilizationPerDevice(device any, since time.Duration) (map[uint32]any, error) 
```

I can change the function signature in all the underlying devices so the first param isn't device and then put the import back to the original name - NGL I went for the option with smaller impact but I will tidy it up